src/components/PostList/PostsList.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import { getPostsByUserId } from '../../api/loadData';
+import { Post } from '../../types/Post';
+import { Loader } from '../Loader';
+
+type Props = {
+  selectedUserId: number,
+  onSelectPost: (post: Post | null) => void,
+  selectedPost: Post | null,
+};
+
+export const PostsList: React.FC<Props> = ({
+  selectedUserId,
+  onSelectPost,
+  selectedPost,
+}) => {
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isLoadingFinish, setIsLoadingFinish] = useState(false);
```

Comment:
Are they doing the same logic? Why do not use just `isLoading`?

Reply:
Great comments. thank you!